management/server/http/handler/setupkeys.go

```diff
@@ -41,7 +41,7 @@ type SetupKeyRequest struct {
 	Revoked   bool
 }
 
-func NewSetupKeysHandler(accountManager *server.AccountManager, authAudience string) *SetupKeys {
+func NewSetupKeysHandler(accountManager *server.DefaultAccountManager, authAudience string) *SetupKeys {
```

Comment:
I suggest keeping the AccountManager interface here, then we can easily replace it with some other implementation if needed.
It will also be easier to mock in tests
That is the whole purpose of interfaces :)